src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -1,23 +1,20 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Code will be cleaner if you import only the required classes from java.util instead of importing the whole package.

Reply:
+1